README.md

```diff
@@ -26,26 +26,31 @@ DesignGuide requires [Node.js](https://nodejs.org/en/) (includes npm).
 5. Works? If not, contact us on Slack ([#design-guide-general](https://payex.slack.com/messages/C0L3W8B2S/)).
 6. Get to work.
 
+### Set up Git Hooks
+
+- Open a terminal and navigate to root of repository
+- Run `git config core.hooksPath .githooks`
+
 ## Developing
 
 ### Built with
 
-* [React](https://reactjs.org/)
-* [Less](http://lesscss.org/)
+- [React](https://reactjs.org/)
```

Comment:
These `*` to `-` changes aren't exactly related to git hooks, but 🤷‍♂️.

Reply:
Yeah.. md-linter started whining so thought I'd hit two birds with one stone

Reply:
![last ned](https://user-images.githubusercontent.com/26485094/49024255-5e594880-f199-11e8-80c4-0c8e8a483366.jpg)